hermes/probe/read/read_test.go

```diff
@@ -87,12 +73,14 @@ func TestReadFile(t *testing.T) {
 	ctx := context.Background()
 	client := fakegcs.NewClient()
 	bucketName := "test_bucket_probe0"
```

Comment:
Nit: this is called `bucket` in other places. I would recommend making it consistent.

Reply:
changed